imports/client/components/PuzzlePage.jsx

```diff
@@ -684,12 +684,10 @@ class PuzzlePageMetadata extends React.Component {
     const isAdministrivia = _.findWhere(tags, { name: 'administrivia' });
     const answerComponent = this.props.puzzle.answer ? (
       <span className="puzzle-metadata-answer">
-        Solved:
-        {' '}
         <span className="answer">{this.props.puzzle.answer}</span>
       </span>
     ) : null;
-    const hideViewCount = this.props.puzzle.answer || this.props.subcountersDisabled;
+    const hideViewCount = this.props.subcountersDisabled;
```

Comment:
I was a little surprised by this; generally we disable showing sub counters for solved puzzles at least on the puzzle list page, but I concede that it's weird to immediately hide them on the puzzle page, so I think I support this behavior change